### PR TITLE
Upgrade airbnb config, add install guide

### PR DESCRIPTION
**Pull request by derekstavis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,11 +6,40 @@ Our style guide is based on [Airbnb's](https://github.com/airbnb/javascript). Di
...
+## Installing
+
+The rules described in this repository are also available as a eslint
+config package. To install the package and its dependencies:
+
+```shell
+$ npm install --save-dev eslint@4.3.0 \
+                         eslint-plugin-jsx-a11y@5.1.1 \
+                         eslint-plugin-import@2.7.0 \
+                         eslint-plugin-react@7.1.0 \
```
Comment (mccraveiro) [inline README.md]: @derekstavis pagarme-base shouldn't have react dependencies

File: package.json [hunk 2]
```diff
@@ -20,11 +20,7 @@
     "url": "https://github.com/pagarme/javascript-style-guide/issues"
   },
   "homepage": "https://github.com/pagarme/javascript-style-guide#readme",
-  "peerDependencies": {
```
Comment (mccraveiro) [inline package.json]: Why did you remove `peerDependencies`?

Comment (felquis) [inline package.json]: I had to manually install `eslint-plugin-import`

Comment (derekstavis) [inline package.json]: Because it's already declared by the `eslint-config-airbnb-base` package!

Comment (derekstavis) [inline package.json]: Yes @felquis, the instructions are in the readme file.

**Comment by derekstavis** [comment]
Another good discussion to have: Should be bump a major or a minor? When bumping AirBnb configuration version, should we also bump the same as they bumped?

File: README.md [hunk 3]
```diff
@@ -6,11 +6,39 @@ Our style guide is based on [Airbnb's](https://github.com/airbnb/javascript). Di
...
 
-## Semicolons
+## Installing
+
+The rules described in this repository are also available as a eslint
+config package. To install the package and its dependencies:
+
+```shell
+$ npm install --save-dev eslint@4.3.0 \
+                         eslint-plugin-jsx-a11y@5.1.1 \
```
Comment (mccraveiro) [inline README.md]: @derekstavis JSX is also a react dependency

**Comment by mccraveiro** [comment]
@derekstavis IMO it should be a major bump

**Comment by grvcoelho** [comment]
@derekstavis: major bump :)

**Comment by derekstavis** [comment]
Fixed all the issues! After merging I will bump the major :+1:
